### ISSUE TITLE
User collaboration

### ISSUE THREAD

**Issue by hugowetterberg** [hugowetterberg/a-fiend#3]
Easy sharing / importing of alias sets?

**Comment by voxpelli** [comment]
To share between many computers and within small teams, maybe enable putting additional storages in Dropbox (or other shared place in the file system)?

Exporting/importing based on some string based data that one can share over a chat would also be nice.

**Comment by hugowetterberg** [comment]
Yeah, I was thinking along the lines of a data url/ base64 string for direct sharing. And read-only sources loaded by url / file path.
